src/autofill.js

```diff
@@ -1,3 +1,6 @@
+// TODO: this must be injected at page start, not here. Remove it once ready.
+require('./captureDdgGlobals');
```

Comment:
It's here only while developing. I'll remove it and have the native layer inject the file at page start as discussed.

Reply:
Can we avoid merging until this is ready?

Reply:
Yep, that's the plan. There's a warning in the task description as well 🙂.